dev/ci/ci-basic-overlay.sh

```diff
@@ -62,14 +62,14 @@
 : "${Iris_CI_GITURL:=https://gitlab.mpi-sws.org/FP/iris-coq}"
```

Comment:
When using a fixed commit for lambdarust we could also specify the dependencies here instead of parsing opam files, should we do that?